### DISCUSSION TITLE
Side by Side both ways

### DISCUSSION THREAD

**Discussion by fuzzyfaraway** [thygate/stable-diffusion-webui-depthmap-script#105] [Ideas]
**Request:** I would love to see simultaneous rendering of _both_ parallel and cross-eye SBS views.

I can't parallel view to save my life, so I really do appreciate the recent addition of the setting switch to enable rendering cross-eye SBS. I was using Photoshop to change the parallel views to cross-eye. But it would be absolutely terrific to get all three views, parallel, cross-eye and anaglyph, at one time for when sharing with people, whatever their preferred method is, it would be available.

Thanks very much for developing this extension. It works great and I love creating 3D views!

**Comment by finefin** [comment]
+1 for parallel/cross-eye view toggle. I can do both but parallel is much harder.

**Comment by WorldofDepth** [comment]
AlternativeIy I would suggest an option to save individual left and right stereo views. Then folks can combine them as parallel or cross or three-in-a-row universal as they like.

**Comment by finefin** [comment]
I just found out that you can you can enable "Flip Left/Right in SBS stereo images" in the settings!

**Reply by fuzzyfaraway** [reply]
Yes, that's kind of what my starting request was about. With that setting you still have to choose either parallel or crosseye view, but not get both at once. Use case example: If you want to post, say, on Instagram for stereography fans (and there are some on there), it is common to post both parallel and crosseye as well as anaglyph versions. Being able to produce SD renders in all three at once would be fantastic.

And, after starting this reply I went to double-check things in the extension interface and I noticed that with a recent extension update there are now more choices in a new dropdown menu in the extension itself, but you only get an _either-or_ choice _even more limited than before_. I would really love to have a group of checkboxes to choose any, all or none, making rendering desired views a one-and-done operation.

**Reply by thygate** [reply]
All stereo formats now available at once in newest version.

**Reply by fuzzyfaraway** [reply]
Ah, man! 😁 This squeaky wheel is feelin' the grease! Thank you!
